src/client/Root.react.js

```diff
@@ -8,15 +8,17 @@ import { BrowserRouter } from 'react-router-dom'
 import { SubscriptionClient, addGraphQLSubscriptions } from 'subscriptions-transport-ws'
 
 import App from '../shared/nt-core/App.react'
-import config from '../shared/configs'
 import createStore from '../shared/nt-store/createStore'
 
+const configObj = {
+  server: process.env.BROWSER ? window.__CONFIG__ : require('config').server
+}
```

Comment:
As I told you about `createNetworkInterface` function,

I will remove `configObj` variable then create `createNetworkInterface` with a code below,

```javascript
const networkInterface = createNetworkInterface({
  uri: `/graphql`,
  opts: {
    credentials: 'same-origin'
  },
})
```